deployments/multiple-ragapps/docker-compose.yml

```
@@ -54,9 +54,11 @@ services:
       - KEYCLOAK_ADMIN_PASSWORD=admin
     volumes:
       # Mount example realm data
-      - ./keycloak/example/:/opt/keycloak/data/
+      - ./data/keycloak/example/:/opt/keycloak/data/
```

```suggestion
      - ${PWD}/data/keycloak/example/:/opt/keycloak/data/
```

and how about defining a STATE_DIR env variable (here with `${PWD}/data`)

---

Great idea, i just added it. But it's now cannot use an empty STATE_DIR for now, the keycloak and traefik is required default config and example data to startup.